css/index.css

```diff
@@ -1,144 +1,150 @@
-  
-  */
-  
-  
-  /*======= Create a selector for ALL images in the site to have the following styles: 
-  
-    border-radius: 10px;
-  
-  */
-  
-  .content-row {
-    display: flex;
-  }
-  
-  .content-row div {
-    /*======= create a margin propery.  Set the value to 20px */
-  }
-  
-  .container {
-    /*======= create a width property. Set the value to 600px */
-    margin: 0 auto;
-  }
-  
-  .container header nav a {
-    margin: 0 10px;
-    border: 1px solid lightgray;
-    padding: 20px;
-    display: inline-block;
-    border-radius: 20px;
-    color: white;
-    text-decoration: none;
-    margin-bottom: 20px;
-     /*======= create a background property. Set the value to #F87B99 */
-    
-  }
-  
-  .container header nav a:hover {
-  /*======= create a background property and set it to white.  Then create a color property and set it's value to pink. */
-  }
-  
-  /*======= Create a selector for the about section to have the following styles:
-    margin-top: 20px;
-  */
-  
-  .contact-section {
-    border-top: 1px dashed black;
-    padding: 20px 0;
-  }
-  
-  .contact-section h2 {
-    padding: 0;
-    margin: 0 0 10px 0;
-  }
-  
-  /*======= Create a selector for the address tag in the contact-section class to have the following styles:
-     line-height: 2;
-  */
-  
-  footer {
-    width: 100%;
-    background: #F87B99;
-    border-top: 1px dashed white;
-  }
-  
-  footer .footer-content { 
-    margin: 0 auto; 
-    /*======= create a width property. Set the value to 600px */
-    
-  }
-  
-  footer .footer-content h3 {
-    padding: 0;
-    margin: 20px 0;
-  }
-  
-  footer nav {
-    display: flex;
-  }
-  
-  /*======= Create a selector for the anchor tag in the footer to have the following styles:
-    margin: 10px;
-    color: black;
-    text-decoration: underline;
-  */
-  
-  
-  footer nav a:first-child {
-    margin-left: 0;
-  }
-  
-  /*======= Create a hover state for the anchor tag in the footer nav to have the following styles:
-    color: white;
-  */
-  
-  footer .footer-content .copyright {
-    padding: 20px 0;
-     /*======= center the text here */
-  }
\ No newline at end of file
+footer .footer-content { 
+  margin: 0 auto; 
+  /*======= create a width property. Set the value to 600px */
+  width: 600%;
+}
+
+footer .footer-content h3 {
+  padding: 0;
+  margin: 20px 0;
+}
+
+footer nav {
+  display: flex;
+}
+
+/*======= Create a selector for the anchor tag in the footer to have the following styles:
+  margin: 10px;
+  color: black;
+  text-decoration: underline;
+*/
+a {
+   margin: 10px;
+   color: black;
+   text-decoration: underline;
+   text-align: center;
+}
+
+footer nav a:first-child {
+  margin-left: 0;
+}
+
+/*======= Create a hover state for the anchor tag in the footer nav to have the following styles:
+  color: white;
+*/
+.footer nav a:hover {
+  color: white;
+}
+
+footer .footer-content .copyright {
+  padding: 20px 0;
+   /*======= center the text here */
+}
```

Comment:
You want to do a text align center, here.

Reply:
Ok done.